src/main/java/org/example/entity/Person.java

```diff
@@ -16,6 +16,15 @@ public String getName() {
         return name;
     }
 
+    public Person(String name, String surname, String phoneNumber, String email) {
+        this.name = name;
+        this.surname = surname;
+        this.phoneNumber = phoneNumber;
+        this.email = email;
+    }
+
+    public Person() {}
```

Comment:
add some default values here, maybe we call all clients withount name John Smith, 
so we create a default user as John Smith all the time.